docker/bootstrap/Dockerfile.percona57

```diff
@@ -11,7 +11,7 @@ RUN apt-key adv --keyserver ha.pool.sks-keyservers.net \
     } | debconf-set-selections && \
     apt-get update && \
     apt-get install -y --no-install-recommends \
-        percona-server-server-5.7 libperconaserverclient18.1-dev && \
+        percona-server-server-5.7=5.7.14* libperconaserverclient18.1-dev && \
```

Comment:
Yeah, looks like it.

Can you please create a separate PR for that and we can ask @enisoc for his opinion on that?

The `lite` images are used within Kubernetes for example. I think it makes sense that we fully rebuild them with the latest MySQL version when we update the Vitess binaries in there.